.gitignore

```diff
@@ -18,4 +18,7 @@ application/logs/*
 /nbproject/private/
 /nbproject/
 
-tmp/*
\ No newline at end of file
+tmp/*
+
+#code coverage report
+tests/coverage
```

Comment:
Ignore `tests/report` instead of `tests/coverage`

Reply:
right. fixed